### ISSUE TITLE
Does not work with Date objects using non-function form.

### ISSUE THREAD

**Issue by dtinth** [substantial/updeep#36]
Example scenario:

```js
var updeep = require('updeep')
var entry = { title: 'Does not work with Date objects using non-function form.' }
```

Using function form works:

```js
> updeep({ created: () => new Date() }, entry)
{ title: 'Does not work with Date objects using non-function form.',
  created: Tue Sep 15 2015 12:38:37 GMT+0700 (ICT) }
```

Using non-function form does not work:

```js
> updeep({ created: new Date() }, entry)
{ title: 'Does not work with Date objects using non-function form.',
  created: {} }
```

**Comment by aaronjensen** [comment]
Fixed in 0.10.1, thanks for the report!

**Comment by dtinth** [comment]
That was quick, thanks!
